sdk/ai/azure-ai-projects/azure/ai/projects/aio/operations/_patch.py

```diff
@@ -0,0 +1,2472 @@
+         AgentsNamedToolChoice Default value is None.
+        :paramtype tool_choice: str or str or ~azure.ai.projects.models.AgentsApiToolChoiceOptionMode or
+         ~azure.ai.projects.models.AgentsNamedToolChoice
+        :keyword response_format: Specifies the format that the model must output. Is one of the
+         following types: str, Union[str, "_models.AgentsApiResponseFormatMode"],
+         AgentsApiResponseFormat Default value is None.
+        :paramtype response_format: str or str or ~azure.ai.projects.models.AgentsApiResponseFormatMode
+         or ~azure.ai.projects.models.AgentsApiResponseFormat
+        :keyword metadata: A set of up to 16 key/value pairs that can be attached to an object, used
+         for storing additional information about that object in a structured format. Keys may be up to
+         64 characters in length and values may be up to 512 characters in length. Default value is
+         None.
+        :paramtype metadata: dict[str, str]
+        :return: ThreadRun. The ThreadRun is compatible with MutableMapping
+        :rtype: ~azure.ai.projects.models.ThreadRun
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+
+        if isinstance(body, dict):  # Handle overload with JSON body.
+            content_type = kwargs.get("content_type", "application/json")
+            response = super().create_run(thread_id, body, content_type=content_type, **kwargs)
+
+        elif assistant_id is not _Unset:  # Handle overload with keyword arguments.
+            response = super().create_run(
+                thread_id,
+                assistant_id=assistant_id,
+                model=model,
+                instructions=instructions,
+                additional_instructions=additional_instructions,
+                additional_messages=additional_messages,
+                tools=tools,
+                stream_parameter=False,
+                stream=False,
+                temperature=temperature,
+                top_p=top_p,
+                max_prompt_tokens=max_prompt_tokens,
+                max_completion_tokens=max_completion_tokens,
+                truncation_strategy=truncation_strategy,
+                tool_choice=tool_choice,
+                response_format=response_format,
+                metadata=metadata,
+                **kwargs,
+            )
+
+        elif isinstance(body, io.IOBase):  # Handle overload with binary body.
+            content_type = kwargs.get("content_type", "application/json")
+            response = super().create_run(thread_id, body, content_type=content_type, **kwargs)
+
+        else:
+            raise ValueError("Invalid combination of arguments provided.")
+
+        # If streaming is enabled, return the custom stream object
+        return await response
+
+    @distributed_trace_async
+    async def create_and_process_run(
+        self,
+        thread_id: str,
+        assistant_id: str,
```

Comment:
Can everything below assistant_id be keyword-only?

Reply:
I would do it below thread_id instead to follow the pattern of create_run.

Reply:
Done

Reply:
Maybe i'm missing it, but I don't see the update. Was it only done with sync?